### PR TITLE
Salaboy api refactoring 3rd

### PR DESCRIPTION
**Pull request by erdemedeiros**

### CODE REVIEW

File: activiti-cloud-services-runtime-bundle/activiti-cloud-services-identity-basic/pom.xml [hunk 1]
```diff
@@ -24,7 +24,9 @@
     </dependency>
     <dependency>
       <groupId>org.activiti</groupId>
-      <artifactId>activiti-identity-basic</artifactId>
+      <artifactId>activiti-spring-identity</artifactId>
+      <!-- @TODO: do deps mgmt -->
+      <version>7-201803-EA-SNAPSHOT</version>
```
Comment (erdemedeiros) [inline activiti-cloud-services-runtime-bundle/activiti-cloud-services-identity-basic/pom.xml]: Remove version

File: activiti-cloud-services-runtime-bundle/activiti-cloud-services-identity-basic/src/test/java/org/activiti/cloud/services/identity/basic/BasicUserGroupManagerIT.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+//import org.springframework.beans.factory.annotation.Autowired;
+//import org.springframework.boot.test.context.SpringBootTest;
+//import org.springframework.context.annotation.ComponentScan;
+//import org.springframework.test.context.junit4.SpringRunner;
+//
+//import static org.assertj.core.api.Assertions.assertThat;
+//
+//@RunWith(SpringRunner.class)
+//@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+//public class BasicUserGroupManagerIT {
```
Comment (erdemedeiros) [inline activiti-cloud-services-runtime-bundle/activiti-cloud-services-identity-basic/src/test/java/org/activiti/cloud/services/identity/basic/BasicUserGroupManagerIT.java]: @Salaboy what is the status of this test?

File: activiti-cloud-starter-runtime-bundle/src/test/resources/access-control.properties [hunk 3]
```diff
@@ -1,4 +1,23 @@
-activiti.cloud.security.user.testuser.test-app.policy.read=process_pool1,ProcessWithVariables
-activiti.cloud.security.user.testadmin.test-app.policy.read=process_pool1,ProcessWithVariables
-activiti.cloud.security.user.hruser.test-app.policy.write=process_pool1,ProcessWithVariables,SimpleProcess,ProcessWithVariables2,ProcessWithBoundarySignal
-activiti.cloud.security.user.hradmin.test-app.policy.write=process_pool1,ProcessWithVariables,SimpleProcess,ProcessWithVariables2,ProcessWithBoundarySignal
\ No newline at end of file
+activiti.security.policies[0].name=MyPolicy for TestUser
```
Comment (erdemedeiros) [inline activiti-cloud-starter-runtime-bundle/src/test/resources/access-control.properties]: I like the new way of describing policies. It's easier to read.

File: activiti-cloud-services-runtime-bundle/activiti-cloud-services-identity-basic/pom.xml (hunk 1 above)
Comment (Salaboy) [reply activiti-cloud-services-runtime-bundle/activiti-cloud-services-identity-basic/pom.xml]: @erdemedeiros can you please remove?

File: activiti-cloud-services-runtime-bundle/activiti-cloud-services-identity-basic/src/test/java/org/activiti/cloud/services/identity/basic/BasicUserGroupManagerIT.java (hunk 2 above)
Comment (Salaboy) [reply activiti-cloud-services-runtime-bundle/activiti-cloud-services-identity-basic/src/test/java/org/activiti/cloud/services/identity/basic/BasicUserGroupManagerIT.java]: @erdemedeiros sorry.. please remove.. I've created it but then I realised that we don't need it.

File: activiti-cloud-starter-runtime-bundle/src/test/resources/access-control.properties (hunk 3 above)
Comment (Salaboy) [reply activiti-cloud-starter-runtime-bundle/src/test/resources/access-control.properties]: yeah! there are a lot of things to improve.. but this new approach simplified a lot of parsing code

File: activiti-cloud-services-runtime-bundle/activiti-cloud-services-identity-basic/pom.xml (hunk 1 above)
Comment (erdemedeiros) [reply activiti-cloud-services-runtime-bundle/activiti-cloud-services-identity-basic/pom.xml]: done

File: activiti-cloud-services-runtime-bundle/activiti-cloud-services-identity-basic/src/test/java/org/activiti/cloud/services/identity/basic/BasicUserGroupManagerIT.java (hunk 2 above)
Comment (erdemedeiros) [reply activiti-cloud-services-runtime-bundle/activiti-cloud-services-identity-basic/src/test/java/org/activiti/cloud/services/identity/basic/BasicUserGroupManagerIT.java]: Moved to Activiti/activiti-spring-identity

**Pull request merged**